solr/solr-ref-guide/antora.yml

```diff
@@ -40,16 +40,17 @@ asciidoc:
     # 'patch-version' is the 'z' part of x.y.z semantic version
     page-solr-javadocs: 'https://solr.apache.org/docs/10_0_0'
     solr-javadocs: 'https://solr.apache.org/docs/10_0_0'
-    lucene-javadocs: 'https://lucene.apache.org/core/9_4_0'
+    lucene-javadocs: 'https://lucene.apache.org/core/9_4_2'
```

Comment:
There is a note about this file not being manually edited:

```
# This file is autogenerated by the "setOfficialAntoraYaml" Gradle task, please do not edit manually.
# Do not run this task outside of the release process on a release branch (branch_9_0, branch_10_2, etc.).
# Feel free to run the task on unreleased branches (main, branch_9x, branch_10x) to update the dependency versions,
# but there is usually no need besides more up-to-date nightly builds of the ref-guide.
```

was this manually updated or did the task do it?

Reply:
Actually I found it strange that this was checked in at all, and as you can see it was somewhat out of date already. It is auto generated from template during build, so this time I copied it from the build/site folder after I ran `gradle solr:solr-ref-guide:buildLocalSite`. I wonder @HoustonPutman / @cpoerschke if there would be a way to auto update this file during a build, so that it would be flagged for commit.

It is not super critical if it is out of date, but the reason I updated it was due to IntelliJ's built-in asciidoc plugin complaining about my new variable not being defined, so it obviously reads this file.